backend/infrahub/graphql/queries/diff/tree.py

```diff
@@ -100,6 +100,7 @@ class DiffNode(DiffSummaryCounts):
     conflict = Field(ConflictDetails, required=False)
     contains_conflict = Boolean(required=True)
     last_changed_at = DateTime(required=False)
+    parent_node = String(required=False)
```

Comment:
is this enough information for the front-end? I would think that they would need the parent relationship name/label as well, but I'm not sure

Reply:
I'll merge as it for now but I'll check with the frontend team tomorrow morning